server/socketCommands.ts

```diff
@@ -16,8 +21,21 @@ const socketCommands = (io: Server)=>{
     const rooms = io.sockets.adapter.rooms
     const roomData: Array<Room> = []
```

Comment:
Probably should keep this consistent. We should always do Room[] or Array<Room>. Personally think Room[] is easier to write so my vote is on that.